src/app/public/modules/progress-indicator/progress-indicator.component.ts

```diff
@@ -114,10 +152,11 @@ export class SkyProgressIndicatorComponent implements OnInit, AfterContentInit,
     this._activeIndex = newIndex;
   }
 
-  private ngUnsubscribe = new Subject();
+  private ngUnsubscribe = new Subject<void>();
 
   private _activeIndex: number;
   private _displayMode: SkyProgressIndicatorDisplayMode;
+  private _hasFinishButton: boolean;
   private _isPassive: boolean;
   private _messageStream = new Subject<SkyProgressIndicatorMessage | SkyProgressIndicatorMessageType>();
   private _startingIndex: number;
```

Comment:
Responded above, with a question.

Reply:
Should these properties all be on their own line?